src/lib/components/UserBig.svelte

```diff
@@ -0,0 +1,101 @@
+<script lang="ts">
```

Comment:
I guess this could be a subset of the <User /> component, but I felt it might actually complicate it unnecessarily